src/ui/FRWComponent/LandingPages/ImportComponents/Googledrive.tsx

```diff
@@ -42,9 +43,12 @@ const Googledrive = ({ setErrorMessage, setShowError }) => {
 
     try {
       const accounts = await wallets.loadBackupAccounts();
+
+      localStorage.setItem('backupAccounts', JSON.stringify(accounts));
+
       if (accounts.length > 0) {
         history.push({
-          pathname: '/import/google',
+          pathname: '/welcome/accountimport/google',
           state: {
             accounts: accounts,
           },
```

Comment:
I'd prefer you told the view containing this component to navigate. Hard coding a route makes it hard to re-use components.